ext/RMagick/rmagick.c

```diff
@@ -52,7 +52,7 @@ Magick_colors(VALUE class)
     exception = AcquireExceptionInfo();
 
     color_info_list = GetColorInfoList("*", &number_colors, exception);
-    CHECK_EXCEPTION()
+    CHECK_EXCEPTION();
```

Comment:
@Watson1978 `CHECK_EXCEPTION()` is already defined with a semicolon, so won't this result in:

```c
rm_check_exception(exception, NULL, RetainOnError);;
```

Reply:
Oh, nevermind. I just noticed that you changed the `define`, too.